romsel_aktheme/arm9/source/common/bootstrapconfig.cpp

```diff
@@ -420,7 +428,447 @@ int BootstrapConfig::launch()
 		.option("NDS-BOOTSTRAP", "CARDENGINE_CACHED", _ceCached)
 		.option("NDS-BOOTSTRAP", "FORCE_SLEEP_PATCH", _forceSleepPatch);
 
-	if (_configSavedHandler)
+	if (_configSavedHandler) {
 		_configSavedHandler();
+	}
+	
+	if (_useWideScreen) {
```

Comment:
todo: widescreen is removed now?

Reply:
The widescreen file is no longer renamed. TWiLight Menu++ is now stretched inwards with the widescreen TWL_FIRM file

Reply:
for reference: https://github.com/DS-Homebrew/TWiLightMenu/commit/6d61dc9ca97c8fee73a19b41fa3dfe0f25d36081

need to check if cosLerp effect can be applied to gdi top background. if